heimdall-core/src/main/java/br/com/conductor/heimdall/core/trace/Trace.java

```diff
@@ -312,22 +312,23 @@ private void writeTrace() throws JsonProcessingException {
 
 	private void printInLogger(Logger logger) throws JsonProcessingException {
 		ObjectMapper mapper = mapper();
+		String message = mapper().writeValueAsString(this);
```

Comment:
`mapper().writeValueAsString(this);` can not be used if else-blocks is invoked, causing a useless processing. Please, keep the previous code.

Reply:
ok